go/README.md

```diff
@@ -2,7 +2,7 @@
 
 This example will show you how to test and deploy Go (Golang) code to IronWorker.
 
-**NOTE**: Be sure you've followed the base [getting started instructions on the top level README](https://github.com/iron-io/dockerworker).
+**NOTE**: Before getting started, make sure your iron.json file is in the root of the project directory. Be sure you've followed the base [getting started instructions on the top level README](https://github.com/iron-io/dockerworker).
```

Comment:
not necessarily true I don't think, I'd rather link to http://dev.iron.io/worker/reference/configuration/#about_the_scheme for `iron.json` instructions because those will stay up to date naturally and offer some other options. a good catch though